src/main/java/de/hsrm/vegetables/my_food_coop_service/services/DisposeService.java

```diff
@@ -0,0 +1,52 @@
+package de.hsrm.vegetables.my_food_coop_service.services;
+
+import de.hsrm.vegetables.my_food_coop_service.model.DisposedItem;
+import de.hsrm.vegetables.my_food_coop_service.model.VatDetailItem;
+import de.hsrm.vegetables.my_food_coop_service.repositories.DisposedRepository;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+@RequiredArgsConstructor(onConstructor = @__({@Autowired}))
```

Comment:
That's unnecessary, as there are no autowired class fields

Reply:
but in getVatDetails() these collector is used

Reply:
@Service
@RequiredArgsConstructor(onConstructor = @__({@Autowired}))
can be deleted i would agree

Reply:
The @Service needs to stay, but line 17 needs to be removed